tephra-core/src/main/java/org/apache/tephra/TxConstants.java

```diff
@@ -369,6 +369,12 @@
      * Interval in seconds to schedule prune run.
      */
     public static final String PRUNE_INTERVAL = "data.tx.prune.interval";
+
+    /**
+     * Interval in seconds to schedule flush of prune table entries to store.
+     */
+    public static final String PRUNE_FLUSH_INTERVAL = "data.tx.prune.flush.interval";
```

Comment:
In the first commit, it was not being used anywhere. Since it is being used now, it is okay.